src/components/announcer/Announcer.js

```diff
@@ -6,157 +6,110 @@
-
-import { withStyles } from "@material-ui/core/styles";
-
-const variantIcon = {
-    success: CheckCircleIcon,
-    warning: WarningIcon,
-    error: ErrorIcon,
-    info: InfoIcon,
-};
+import { Snackbar, useTheme, Typography } from "@material-ui/core";
+import Alert from "@material-ui/lab/Alert";
+
+function getTextColor(theme, variant) {
+    let color;
+    switch (variant) {
+        case Error:
+            color = theme.palette.error.contrastText;
+            break;
+        case SUCCESS:
+            color = theme.palette.success.contrastText;
+            break;
+        case WARNING:
+            color = theme.palette.warning.contrastText;
+            break;
+        default:
+            color = theme.palette.info.contrastText;
+    }
 
-const styles1 = (theme) => ({
-    success: {
-        backgroundColor: palette.darkGreen,
-    },
-    error: {
-        backgroundColor: palette.red,
-    },
-    info: {
-        backgroundColor: palette.blue,
-    },
-    warning: {
-        backgroundColor: palette.orange,
-    },
-    icon: {
-        fontSize: 20,
-    },
-    iconVariant: {
-        opacity: 0.9,
-        marginRight: theme.spacing(1),
-    },
-    message: {
-        display: "flex",
-        alignItems: "center",
-    },
-    margin: {
-        margin: theme.spacing(1),
-    },
-});
+    return color;
+}
 
 function MySnackbarContent(props) {
-    const { classes, message, onClose, variant, customAction } = props;
-    const Icon = variantIcon[variant];
-
+    const { message, onClose, variant, action } = props;
+    const theme = useTheme();
     return (
-        <SnackbarContent
-            className={classNames(classes[variant], classes.margin)}
-            aria-describedby="client-snackbar"
-            message={
-                <span id="client-snackbar" className={classes.message}>
-                    <Icon
-                        className={classNames(
-                            classes.icon,
-                            classes.iconVariant
-                        )}
-                    />
-                    {message}
-                </span>
-            }
-            action={[
-                customAction ? customAction() : null,
-                <IconButton
-                    key="close"
-                    aria-label="Close"
-                    color="inherit"
-                    className={classes.close}
-                    onClick={() => {
-                        console.log("close");
-                        onClose();
-                    }}
-                >
-                    <CloseIcon className={classes.icon} />
-                </IconButton>,
-            ]}
-        />
+        <Alert
+            elevation={6}
+            variant="filled"
+            severity={variant}
+            onClose={onClose}
+            action={action}
+            style={{ color: getTextColor(theme, variant) }}
+        >
+            {message}
```

Comment:
I think calling `{message}` like this requires `message` to be a `string`.
If we used `<Message />` here, then the message text could come from a `getMessage` call.

Although I think keeping a string `{message}` is more flexible, since it gives us the option of directly passing some string message from a backend response directly to the announcer.

Reply:
Hmm am not sure `{message}` vs `<Message /> is the issue.  A lot Material UI example uses `{}` syntax for nodes / DOM. See code for - https://material-ui.com/components/drawers/#responsive-drawer

Reply:
Oh, I forgot our `getMessage` wrapper returns a React tag: https://github.com/cyverse-de/ui-lib/blob/69c1cc5653b8d3c36e65eff36ce74bf11242f16b/src/util/I18NWrapper.js#L29-L34
So that should work fine here, as far as I can tell.
I'm not familiar with the issue that @ash3rz is referring to, but maybe she is right that you can't use an object like this as a `key`.

Reply:
The only issue I was talking about is that the `message` prop won't accept `getMessage` because it returns an object. So any time we want to use this component, we have to use `formatMessage` which requires more set up.  It's just a small inconvenience, not a huge deal.